test/geometry/transform/test_imgwarp.py

```diff
@@ -11,6 +11,16 @@
 from kornia.utils.helpers import _torch_inverse_cast
 
 
+class DummyNNModule(torch.nn.Module):
```

Comment:
@johnnv1 i think as post meta task from this we could exploring to prototype an AutoModule that basically given a function parsed via `inspect` to automatically generate Modules. This might requires creating `InputTensor` / `OutputTensor`  alias types.

Reply:
we can work on it